stock_pack_operation_quick_lot/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==============================
+Stock Pack Operation Quick Lot
+==============================
+
+This module allows set pack operation Lot Name and End of Life Date directly
+on picking operations
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Inventory > Operations > All transfer
+#. Select a Ready picking or create and Mark as Todo
+#. Set Lot Name [and date] in Operations tab
```

Comment:
I'm not able to select any lot in this point.

![2018-03-09_18-15-40](https://user-images.githubusercontent.com/8736623/37220195-260d48ec-23c6-11e8-88ed-48a35889f61f.png)

Tested in http://3319581-410-b1204b.runbot2.odoo-community.org/web?debug=#id=20&view_type=form&model=stock.picking&menu_id=192&action=266

Reply:
@rafaelbn of course, because it's for creating it on incoming receptions!!

Reply:
@pedrobaeza @carlosdauden shouldn't be hidden for outgoing pickings?

Reply:
There's a technical glitch with that